### PR TITLE
BETA: Register cumcord.is-a.dev

### PR DESCRIPTION
**Pull request by Caiden-P**
Added `cumcord.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/caiden.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Caiden-P",
+        "email": "iididhejejdj@gmail.com"
+    },
+    "record": {
+        "TXT": "."
```
Comment (Stef-00012) [inline domains/caiden.json:7]: TXT records alone do nothing

File: domains/caiden.xyz.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Caiden-P",
+        "email": "iididhejejdj@gmail.com"
+    },
+    "record": {
+        "TXT": ".\"
```
Comment (Stef-00012) [inline domains/caiden.xyz.json:7]: TXT records alone do nothing (also invalid JSON, the `\` escapes the closing `"`)

File: domains/cumcord.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Caiden-P",
+        "email": "iididhejejdj@gmail.com"
+    },
+    "record": {
+        "TXT": "."
```
Comment (Stef-00012) [inline domains/cumcord.json:7]: TXT records alone do nothing

File: domains/lodestone.json [hunk 4]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Caiden-P",
+        "email": "iididhejejdj@gmail.com"
+    },
+    "record": {
+        "TXT": "."
```
Comment (Stef-00012) [inline domains/lodestone.json:7]: TXT records alone do nothing

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Pull request closed without merging**